downstream/modules/analytics/con-review-savings-calculations.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// assembly-automation-savings-planner.adoc
+
+
+[id="con-review-savings-calculations_{context}"]
+
+= Review savings calculations for your automation plans
```

Comment:
Are there any RBAC requirements to access these features?

Reply:
RBAC will be implemented at a later date, so not at the moment. When it does I'll have to come back to these and add prerequisites based on role permissions